middleware/logger.js

```diff
@@ -1,4 +1,4 @@
-const moment = require('moment');
+const moment = require('../client/node_modules/moment');
```

Comment:
never do this. sever code base must be independent from client code base
if moment is to be used in server, must be gone from client package.json